### PR TITLE
SRS Options: Reset all SRS State Values

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
SRS Options: Reset all SRS State Values
See https://github.com/corona-warn-app/cwa-app-tech-spec/pull/186/commits/ffdd7b9d2b96f9e0e3a85fae559420727fbf5a17

## Screenshots
Screenshot 2022-12-06 at 17 01 45

### CODE REVIEW

**Review by puneetmahali1 — APPROVED** [review]
👍

File: src/xcode/ENA/ENA/Source/Developer Menu/DMSRSPrechecks/DMSRSOptionsViewModel.swift [hunk 1]
```diff
@@ -61,15 +61,27 @@ final class DMSRSOptionsViewModel {
...
-				})
+				}
+			)
+		case .resetSRSStateValues:
+			return DMButtonCellViewModel(
+				text: "SRS State Values",
+				textColor: .white,
+				backgroundColor: .enaColor(for: .buttonDestructive),
+				action: { [store] in
+					store.mostRecentKeySubmissionDate = nil
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Developer Menu/DMSRSPrechecks/DMSRSOptionsViewModel.swift:80]: Shouldn't it be SRS OTP expiration date?

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Please check my comment

File: src/xcode/ENA/ENA/Source/Developer Menu/DMSRSPrechecks/DMSRSOptionsViewModel.swift (hunk 1 above)
Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Developer Menu/DMSRSPrechecks/DMSRSOptionsViewModel.swift:80]: Yes, that the next line ;)

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**